src/BallInf.jl

```diff
@@ -134,46 +142,57 @@ It is defined as the norm of the enclosing ball, of the given
 ### Output
 
 A real number representing the norm.
+
+### Notes
+
+The norm of an infinity ball is defined as the norm of the enclosing ball, of
+the given ``p``-norm, of minimal volume.
 """
 function norm(B::BallInf, p::Real=Inf)::Real
     return maximum(map(x -> norm(x, p), vertices_list(B)))
 end
 
 """
-    radius(B::BallInf, [p])
+    radius(B::BallInf, [p]::Real=Inf)
 
 Return the radius of a ball in the infinity norm.
 
-It is defined as the radius of the enclosing ball of the given ``p``-norm of
-minimal volume with the same center.
-
 ### Input
 
-- `B` -- a ball in the infinity norm
+- `B` -- ball in the infinity norm
 - `p` -- (optional, default: `Inf`) norm
 
 ### Output
 
 A real number representing the radius.
+
+### Notes
+
+The radius is defined as the radius of the enclosing ball of the given
+``p``-norm of minimal volume with the same center.
 """
-radius(B::BallInf, p::Real=Inf) = (p == Inf) ? B.radius : norm(fill(B.radius, dim(B)), p)
+radius(B::BallInf, p::Real=Inf) =
+    (p == Inf) ? B.radius : norm(fill(B.radius, dim(B)), p)
 
 """
-    diameter(B::BallInf, [p])
+    diameter(B::BallInf, [p]::Real=Inf)
 
 Return the diameter of a ball in the infinity norm.
 
-It corresponds to the maximum distance between any two elements of the set.
-Equivalently, it is the diameter of the enclosing ball of the given ``p``-norm
-of minimal volume with the same center.
-
 ### Input
 
-- `B` -- a ball in the infinity norm
+- `B` -- ball in the infinity norm
 - `p` -- (optional, default: `Inf`) norm
 
 ### Output
 
 A real number representing the diameter.
+
+### Notes
+
+The diameter is defined as the maximum distance in the given ``p``-norm between
```

Comment:
we can put the definition on top as the second sentenc. the equivalent definition can be left here in the "Notes" paragraph if you like